app/views/static_pages/home.html.erb

```diff
@@ -1,11 +1,28 @@
 <% provide :title, t(".title") %>
-<div class="center jumbotron">
-  <h1><%= t ".welcome"%></h1>
-  <h2>
-    <%= t ".home_homepage" %>
-    <%= link_to t(".home_ror_tutorial"), Settings.home_page %>
-    <%= t ".home_sample_app" %>
-  </h2>
-  <%= link_to t(".home_signup"), signup_path, class: "btn btn-lg btn-primary" %>
-</div>
-<%= link_to image_tag("rails.svg", alt: "Rails logo", width: Settings.width), Settings.home_page %>
+<% if logged_in? %>
+  <div class="row">
+    <aside class="col-md-4">
+      <section class="user_info">
+        <%= render "shared/user_info" %>
+      </section>
+      <section class="micropost_form">
+        <%= render "shared/micropost_form" %>
+      </section>
+    </aside>
+    <div class="col-md-8">
+      <h3><%= t(".feed") %></h3>
+      <%= render "shared/feed" %>
+    </div>
+  </div>
+<% else %>
```

Comment:
nếu người dùng đăng nhập thì sẽ in ra thông tin người dùng, bài viết, và form để tạo bài viết